0x48piraj/app.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, render_template, flash, request
+from wtforms import Form, TextField, TextAreaField, validators, StringField, SubmitField
+import sys
+import json
+import os
+import shutil
+from os import path
+
+
+#import urllib #Deprecated in py3
```

Comment:
Remove unnecessary comments. So we use comments to describe the code so that anyone can understand what this code is doing. Know more about it [here](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/)

Reply:
Ok, I removed the comments and indented the code. :-)